src/library/Main.java

```diff
@@ -14,225 +14,223 @@
-	}
-
-
-	private static void RETURN_BOOK() {
-		new ReturnBookUI(new rETURN_bOOK_cONTROL()).RuN();		
-	}
-
-
-	private static void FIX_BOOKS() {
-		new FixBookUI(new FixBookControl()).fixBook();		
-	}
-
-
-	private static void INCREMENT_DATE() {
-		try {
-			int days = Integer.valueOf(input("Enter number of days: ")).intValue();
-			CAL.incrementDate(days);
-			LIB.checkCurrentLoans();
-			output(SDF.format(CAL.getDate()));
-			
-		} catch (NumberFormatException e) {
-			 output("\nInvalid number of days\n");
-		}
-	}
-
-
-	private static void ADD_BOOK() {
-		
-		String AuThOr = input("Enter author: ");
-		String TiTlE  = input("Enter title: ");
-		String CaLl_NuMbEr = input("Enter call number: ");
-		Book BoOk = LIB.addBook(AuThOr, TiTlE, CaLl_NuMbEr);
-		output("\n" + BoOk + "\n");
-		
-	}
-
-	
-	private static void ADD_MEMBER() {
-		try {
-			String LaSt_NaMe = input("Enter last name: ");
-			String FiRsT_NaMe  = input("Enter first name: ");
-			String EmAiL_AdDrEsS = input("Enter email address: ");
-			int PhOnE_NuMbEr = Integer.valueOf(input("Enter phone number: ")).intValue();
-			Member MeMbEr = LIB.addMember(LaSt_NaMe, FiRsT_NaMe, EmAiL_AdDrEsS, PhOnE_NuMbEr);
-			output("\n" + MeMbEr + "\n");
-			
-		} catch (NumberFormatException e) {
-			 output("\nInvalid phone number\n");
-		}
-		
-	}
-
-
-	private static String input(String prompt) {
-		System.out.print(prompt);
-		return IN.nextLine();
-	}
-	
-	
-	
-	private static void output(Object object) {
-		System.out.println(object);
-	}
-
-	
+    
+    private static Scanner in;
+    private static Library lib;
+    private static String menu;
+    private static Calendar cal;
+    private static SimpleDateFormat sdf;
+    
+    
+    private static String getMenu() {
+        StringBuilder sb = new StringBuilder();
+        
+        sb.append("\nLibrary Main Menu\n\n")
+          .append("  M  : add member\n")
+          .append("  LM : list members\n")
+          .append("\n")
+          .append("  B  : add book\n")
+          .append("  LB : list books\n")
+          .append("  FB : fix books\n")
+          .append("\n")
+          .append("  L  : take out a loan\n")
+          .append("  R  : return a loan\n")
+          .append("  LL : list loans\n")
+          .append("\n")
+          .append("  P  : pay fine\n")
+          .append("\n")
+          .append("  T  : increment date\n")
+          .append("  Q  : quit\n")
+          .append("\n")
+          .append("Choice : ");
+          
+        return sb.toString();
+    }
+
+
+    public static void main(String[] args) {        
+        try {            
+            in = new Scanner(System.in);
+            lib = Library.getInstance();
+            cal = Calendar.getInstance();
+            sdf = new SimpleDateFormat("dd/MM/yyyy");
+    
+            for (Member m : lib.listMembers()) {
+                output(m);
+            }
+            output(" ");
+            for (Book b : lib.listBooks()) {
```

Comment:
With these 'single letter variables' sprinkled throughout this section, you may want to see if you can give them more meaningful names. The ones I can see are "m", "b", "e" and "c".

Reply:
Resolution: https://github.com/9Mcgusto6/ITC205-Team17/pull/5/commits/b7072f57079a2158aea67b85387698da1fa40995